Resources/config/routing_rest.yml

```diff
@@ -1,15 +1,13 @@
 recommendationBundle_getContent:
-    path: /ez_recommendation/v1/content/{contentIdList}/{responseType}
+    path: /ez_recommendation/v1/content/{contentIdList}
```

Comment:
It concerns me that you have something that looks like path prefix defined as a path. 
Version of what `v1` refers to?
What is the full URI of this endpoint, looking from client side?

Reply:
The full URI is `/api/ezp/v2/ez_recommendation/v1/`, I will simplify it but not in this PR